tests/integration/test_schema_protobuf.py

```diff
@@ -56,11 +56,61 @@ async def test_protobuf_schema_compatibility(registry_async_client: Client, trai
 
     original_schema = trim_margin(original_schema)
 
+    # Same schema with different whitespaces to see API equality comparison works
+    original_schema_with_whitespace = trim_margin(
+        """
+            |syntax = "proto3";
+            |
+            |package a1;
+            |
+            |
+            |message TestMessage {
+            |    message Value {
+            |        string str2 = 1;
+            |      int32 x = 2;
+            |    }
+            |  string test = 1;
+            |      .a1.TestMessage.Value val = 2;
+            |}
+            |"""
+    )
+
     res = await registry_async_client.post(
         f"subjects/{subject}/versions{trail}", json={"schemaType": "PROTOBUF", "schema": original_schema}
     )
     assert res.status_code == 200
     assert "id" in res.json()
+    original_id = res.json()["id"]
+
+    res = await registry_async_client.post(
```

Comment:
Separate test for the normalization as this test function is about compatibility.